src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -15,15 +22,17 @@ const ScenarioSidebar: React.FC = () => {
           </p>
         </div>
         <div>
-          <p className="text-xs text-gray-500">Last Update: March 25, 2020</p>
+          <p className="text-xs text-gray-500">Last Update:</p>
```

Comment:
I'm not 100% sure how Firestore returns date times so you'll likely need to do some formatting here.  The `DateFormats` file may be of service:  https://github.com/Recidiviz/covid19-dashboard/blob/94e5cafa080eeebc63aadd032f1a94643f8a403a/src/design-system/DateFormats.tsx